example/manifest/manifest.go

```diff
@@ -0,0 +1,395 @@
+		ID:   issuerDID.String(),
+		Name: "CA DMV",
+		Styles: &rendering.EntityStyleDescriptor{
+			Background: &rendering.ColorResource{Color: "#FFFFFF"},
+			Text:       &rendering.ColorResource{Color: "#000000"},
+		},
+	}); err != nil {
+		return nil, err
+	}
+
+	// Set the output descriptor, which is the resulting credential
+	if err := builder.SetOutputDescriptors([]manifest.OutputDescriptor{
+		{
+			ID:          uuid.NewString(),
+			Schema:      licenseSchemaID,
+			Name:        "Drivers License",
+			Description: "A CA Drivers License",
+			Styles: &rendering.EntityStyleDescriptor{
+				Background: &rendering.ColorResource{Color: "#FFFFFF"},
+				Text:       &rendering.ColorResource{Color: "#000000"},
+			},
+		},
+	}); err != nil {
+		return nil, err
+	}
+
+	// Set the input descriptors, which are the inputs required to issue the resulting credential
+	if err := builder.SetPresentationDefinition(exchange.PresentationDefinition{
+		ID: uuid.NewString(),
+		InputDescriptors: []exchange.InputDescriptor{
+			{
+				ID:      "firstName",
+				Name:    "First Name",
+				Purpose: "Provide your first name required to issue your drivers license",
+				Format: &exchange.ClaimFormat{
+					JWTVC: &exchange.JWTType{Alg: []crypto.SignatureAlgorithm{crypto.EdDSA}},
+				},
+				Constraints: &exchange.Constraints{
+					SubjectIsIssuer: exchange.Preferred.Ptr(),
+					Fields: []exchange.Field{
+						{
+							Path:     []string{"$.credentialSubject.firstName"},
+							Optional: false,
+						},
+					},
+				},
+			},
+			{
+				ID:      "lastName",
+				Name:    "Last Name",
+				Purpose: "Provide your last name required to issue your drivers license",
+				Format: &exchange.ClaimFormat{
+					JWTVC: &exchange.JWTType{Alg: []crypto.SignatureAlgorithm{crypto.EdDSA}},
+				},
+				Constraints: &exchange.Constraints{
+					SubjectIsIssuer: exchange.Preferred.Ptr(),
+					Fields: []exchange.Field{
+						{
+							Path:     []string{"$.credentialSubject.lastName"},
+							Optional: false,
+						},
+					},
+				},
+			},
+			{
+				ID:      "dateOfBirth",
+				Name:    "Date of Birth",
+				Purpose: "Provide your date of birth required to issue your drivers license",
+				Format: &exchange.ClaimFormat{
+					JWTVC: &exchange.JWTType{Alg: []crypto.SignatureAlgorithm{crypto.EdDSA}},
+				},
+				Constraints: &exchange.Constraints{
+					SubjectIsIssuer: exchange.Preferred.Ptr(),
+					Fields: []exchange.Field{
+						{
+							Path:     []string{"$.credentialSubject.dateOfBirth"},
+							Optional: false,
+						},
+					},
+				},
+			},
+		},
+	}); err != nil {
+		return nil, err
+	}
+
+	// Set the claim format which we'll accept as inputs
+	if err := builder.SetClaimFormat(exchange.ClaimFormat{
+		JWTVC: &exchange.JWTType{Alg: []crypto.SignatureAlgorithm{crypto.EdDSA}},
+	}); err != nil {
+		return nil, err
+	}
+
+	return builder.Build()
+}
+
+// Prepare a credential which is required to fill out the credential manifest's application
```

Comment:
nit:

// Prepare a credential which is required to fill out the credential manifest's application's input descriptor's requirements

Reply:
tyty